samples/subsys/display/lvgl/src/main.c

```diff
@@ -77,7 +77,8 @@ int main(void)
 	}
 #endif
 
-	if (IS_ENABLED(CONFIG_LV_Z_POINTER_KSCAN)) {
+	if (IS_ENABLED(CONFIG_LV_Z_POINTER_KSCAN) ||
+	    IS_ENABLED(CONFIG_DT_HAS_ZEPHYR_INPUT_LVGL_POINTER_ENABLED)) {
```

Comment:
use `INPUT_LVGL_POINTER` here, normally those `DT_HAS` should only be used for dependencies